enterprise/backend/src/metabase_enterprise/caching/api.clj

```diff
@@ -0,0 +1,88 @@
+(ns metabase-enterprise.caching.api
+  (:require
+   [compojure.core :refer [GET]]
+   [java-time.api :as t]
+   [metabase.api.common :as api]
+   [metabase.api.routes.common :refer [+auth]]
+   [metabase.util.cron :as u.cron]
+   [metabase.util.malli.schema :as ms]
+   [toucan2.core :as t2]))
+
+(api/defendpoint GET "/"
+   "Return cache configuration."
+  [:as {{:strs [model collection]
+         :or   {model "root"}}
+        :query-params}]
+  {model      [:set {:decode/string (fn [x] (cond (set? x)    x
+                                                  (vector? x) (set x)
+                                                  (some? x)   #{x}))}
+               [:enum "root" "database" "collection" "dashboard" "question"]]
```

Comment:
does it accept `dashcard` too? in the product doc I see we can config caching strategy for them.

Reply:
https://www.figma.com/file/x87hHuf4RabjMBuyV2Lq7c/caching-arc-low-fi-explorations?type=design&node-id=118-926&mode=design&t=W7sWFUIPt6Jfwv4k-4

this is what I'm talking about

Reply:
and also, I don't think we can config cache for a "collection" right? at least as of now.

Reply:
> *arakaki*
> Um… I don’t think it makes sense… A dashcard is like a question position in a dashboard. Users already can configure caching for dashboards and questions

I'm pretty sure figma shows "questions inside a dashboard" meaning configuration for a card inside that dashboard. But hey, Figma mockup was updated a little bit since I last looked at it, need to think about access pattern here, hmm...

> and also, I don't think we can config cache for a "collection" right? at least as of now.

Why not? It's the same interface and we agreed on it in a tech doc and I'm pretty sure it makes a lot of sense. @luizarakaki any thoughts on this?

Reply:
on collection: this is where I take it from https://www.notion.so/metabase/Rework-cache-setup-and-management-db37ccc280db4da484c15770bd18f0bd?pvs=4#7460c0b5537f43339fe1ac354badc60b